extensions/amp-analytics/integrating-analytics.md

```diff
@@ -26,7 +26,7 @@ Before you can add your analytics service to AMP HTML runtime, you may need to:
         1. ```"vars": {}``` for additional default variables.
         1. ```"requests": {}``` for requests that your service will use.
         1. ```"optout":``` if needed.  We currently don't have a great opt-out system, so please reach out to help us design one that works well for you.
-    1. If you are using iframe transport, add a new block to ANALYTICS_IFRAME_TRANSPORT_CONFIG in vendors.js containing ```"transport": { "iframe": *url* }```
+    1. If you are using iframe transport, add a new block to ANALYTICS_IFRAME_TRANSPORT_CONFIG in iframe-transport-vendors.js containing ```"transport": { "iframe": *url* }```
```

Comment:
Change link on line 35 to:
https://github.com/ampproject/docs/blob/master/content/docs/analytics/analytics-vendors.md
(change ads_analytics to just analytics)

Reply:
Done